packages/sanity/src/presentation/useDocumentLocations.ts

```diff
@@ -54,7 +54,7 @@ function listen(id: string, fields: string[], store: DocumentStore) {
   }
   const params = {id, draftId: getDraftId(id)}
   return store.listenQuery(query, params, {
-    perspective: 'previewDrafts',
+    perspective: 'drafts',
```

Comment:
not strictly related, but took the liberty of renaming this when I came across it.